tests/openwisp2/settings.py

```diff
@@ -84,6 +84,7 @@
 STATIC_URL = '/static/'
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 OPENWISP_USERS_AUTH_API = True
+OPENWISP_IPAM_API = True
```

Comment:
I might be misunderstanding something, but `OPENWISP_IPAM_API` urls can't be disabled: https://github.com/openwisp/openwisp-ipam/blob/master/openwisp_ipam/api/__init__.py

So, what does this new option do?

Reply:
I share the same concern.

Reply:
I tried to add this extra option to give the developer the freedom to STOP the throttling while developing or testing, But it seems to not work, I am trying to figure out why.

Reply:
This option allows the developer to disable the throttling while development.